docs/process/releasing-updates.md

```diff
@@ -47,7 +47,7 @@ Here's what you should do next:
 To walk through this:
 
  - the script works out the next version from what was previously published, based on the channel
- - you should ensure the `version` in `app/package.json` is set to the new version
+ - you should ensure the `version` in `app/package.json` is set to the new version and follows the [semver format](https://semver.org/) of `major.minor.patch`. If you are releasing to `beta/test`, append `-beta[n]/-test[n]` respectively where `[n]` is a number to be incremented from the previous `beta/test` release e.g `1.1.0-beta1` -> `1.1.0-beta2`
```

Comment:
what do you think about modifying it to look more like this:
```
 - you should ensure the `version` in `app/package.json` is set to the new version and follows the [semver format](https://semver.org/) of `major.minor.patch`. examples:
    - for prod, `1.1.0` -> `1.1.1` or `1.1.13` -> `1.2.0` 
    - for beta, `1.1.0-beta1` -> `1.1.0-beta2` or `1.1.13-beta3` -> `1.2.0-beta1`
    - for test, `1.0.14-test2` -> `1.0.14-test3` or `1.1.14-test3` -> `1.2.0-test1`
```

Reply:
That looks ✨